File: desktop-mend/src/main/java/co/samco/mend4/desktop/output/impl/SystemPrintStreamProviderImpl.java
package co.samco.mend4.desktop.output.impl;

import co.samco.mend4.desktop.output.PrintStreamProvider;

import java.io.PrintStream;

public class SystemPrintStreamProviderImpl implements PrintStreamProvider {
    @Override
    public PrintStream err() {
        return System.err;
    }

    @Override
    public PrintStream out() {
        return System.out;
    }
}

File: desktop-mend/src/main/java/co/samco/mend4/desktop/output/impl/SystemPrintStreamProviderImpl.kt
package co.samco.mend4.desktop.output.impl

import co.samco.mend4.desktop.output.PrintStreamProvider
import java.io.PrintStream

class SystemPrintStreamProviderImpl : PrintStreamProvider {
    override fun err(): PrintStream = System.err

    override fun out(): PrintStream = System.out
}